docs/source/how-to.rst

```diff
@@ -89,4 +89,34 @@ order to run:
 
 Moreover, if you already have your read-based coverage files, you can also
 provide it to any of the binners and automatically skip the process of
-creating these read-based coverage files.
\ No newline at end of file
+creating these read-based coverage files.
+
+.. _scaffolds2bin:
+
+...create scaffolds-to-bin tsv files for DAS Tool
+-------------------------------------------------
+
+Besides of contigs in fasta format, DAS Tool requires a tab separated
+scaffolds to bin file from each metagenomic binner. To create these files,
+you must have the bins from each metagenomic binner in distinct directories.
+In this case, we will use the bins from MaxBin2's output (:file:`maxbin_bins/`).
```

Comment:
I suggest change "In this case" for other connector such "now" because in the line 118 you use it again.